### PR TITLE
lint errors fixed

### PR DESCRIPTION
**Pull request by talhamalik883**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by AmanRaj1608 — APPROVED** [review]
can use this for error handling if this makes sense
```js
try {
  // some code that may throw an error
} catch (err: unknown) {
  if (err instanceof Error) {
    console.error(err);
  }
}
```


or instead of adding eslint-ignore, just add `suppressImplicitAnyIndexErrors` in tsconfig

File: packages/account-abstraction/src/HttpRpcClient.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class HttpRpcClient {
     // await this.initializing
     const hexifiedUserOp = deepHexlify(await resolveProperties(userOp1))
 
-    const jsonRequestData: [UserOperation, string] = [hexifiedUserOp, this.entryPointAddress]
+    // const jsonRequestData: [UserOperation, string] = [hexifiedUserOp, this.entryPointAddress]
```
Comment (livingrockrises) [inline packages/account-abstraction/src/HttpRpcClient.ts:46]: it's not used anywhere? also in the reference aa sdk?

File: packages/account-abstraction/src/HttpRpcClient.ts [hunk 2]
```diff
@@ -87,7 +87,7 @@ export class HttpRpcClient {
   async estimateUserOpGas(userOp1: Partial<UserOperation>): Promise<string> {
     // await this.initializing
     const hexifiedUserOp = deepHexlify(await resolveProperties(userOp1))
-    const jsonRequestData: [UserOperation, string] = [hexifiedUserOp, this.entryPointAddress]
```
Comment (livingrockrises) [inline packages/account-abstraction/src/HttpRpcClient.ts:90]: t's not used anywhere? also in the reference aa sdk?

File: packages/smart-account/src/SmartAccount.ts [hunk 3]
```diff
@@ -862,7 +846,8 @@ class SmartAccount extends EventEmitter {
   async prepareRefundTransaction(
     prepareRefundTransactionDto: PrepareRefundTransactionDto
   ): Promise<FeeQuote[]> {
-    let { version, batchId = 1, chainId } = prepareRefundTransactionDto
+    let { version, chainId } = prepareRefundTransactionDto
+    const { batchId = 1 } = prepareRefundTransactionDto
```
Comment (livingrockrises) [inline packages/smart-account/src/SmartAccount.ts:850]: i am actually removing this from Dtos in other PR

**Review by livingrockrises — COMMENTED** [review]
reviewed. we can aim to remove /* eslint-disable  @typescript-eslint/no-explicit-any */ shortly in next release

**Review by livingrockrises — APPROVED** [review]

File: packages/account-abstraction/src/HttpRpcClient.ts (hunk 1 above)
Comment (talhamalik883) [reply packages/account-abstraction/src/HttpRpcClient.ts:46]: its used in aa sdk but the purpose is to just print request data

File: packages/account-abstraction/src/HttpRpcClient.ts (hunk 2 above)
Comment (talhamalik883) [reply packages/account-abstraction/src/HttpRpcClient.ts:90]: same as above

File: packages/account-abstraction/src/HttpRpcClient.ts (hunk 1 above)
Comment (livingrockrises) [reply packages/account-abstraction/src/HttpRpcClient.ts:46]: ok

**Pull request merged**